raster/src/main/scala/geotrellis/raster/histogram/StreamingHistogram.scala

```diff
@@ -396,25 +437,29 @@ class StreamingHistogram(
     * For each q in qs, all between 0 and 1, find a number
     * (approximately) at the qth percentile.
     */
-  def getPercentileBreaks(qs: Seq[Double]): Seq[Double] = {
-    val data = getCdfIntervals
+  def percentileBreaks(qs: Seq[Double]): Seq[Double] = {
+    val data = cdfIntervals
     qs.map({ q =>
-      val tt = data.dropWhile(_._2._2 <= q).next
-      val (d1, pct1) = tt._1
-      val (d2, pct2) = tt._2
-      val x = (q - pct1) / (pct2 - pct1)
+      if (q == 0.0) minValue()
+      else if (q == 1.0) maxValue()
+      else {
+        val tt = data.dropWhile(_._2._2 <= q).next
+        val (d1, pct1) = tt._1
+        val (d2, pct2) = tt._2
+        val x = (q - pct1) / (pct2 - pct1)
 
-      (1-x)*d1 + x*d2
+        (1-x)*d1 + x*d2
+      }
     })
   }
 
-  def getPercentile(q: Double): Double =
-    getPercentileBreaks(List(q)).head
+  def percentile(q: Double): Double =
+    percentileBreaks(List(q)).head
 
-  def getQuantileBreaks(num: Int): Array[Double] =
-    getPercentileBreaks(List.range(0,num).map(_ / num.toDouble)).toArray
+  def quantileBreaks(num: Int): Array[Double] =
```

Comment:
Can we put some scaladocs that talk about the guarantee that we talked about this morning, that there will be no item in the value set this histogram represents that is outside of the bounds of the quantile breaks returned?